### PR TITLE
Add OpenCensus sample code

### PR DESCRIPTION
**Pull request by mayurkale22**
Node version of https://github.com/GoogleCloudPlatform/golang-samples/pull/646/files.

### CODE REVIEW

**Comment by mayurkale22** [comment]
> Hi, can you provide tests please. Thanks

Done, can you please take a look again. Thanks

**Comment by fhinkel** [comment]
Thanks, we also need the .kokoro config files so the tests will be run.

File: opencensus/metrics-quickstart.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+// GOOGLE_APPLICATION_CREDENTIALS are expected by a dependency of this code
+// Not this code itself. Checking for existence here but not retaining (as not needed)
+if (!projectId || !process.env.GOOGLE_APPLICATION_CREDENTIALS) {
+  throw Error('Unable to proceed without a Project ID');
+}
+
+// The minimum reporting period for Stackdriver is 1 minute.
+const exporter = new StackdriverStatsExporter({
+  projectId: projectId,
```
Comment (fhinkel) [inline opencensus/metrics-quickstart.js]: Is the projectId needed? Our client libraries hat the convention to use the `GOOGLE_APPLICATION_CREDENTIALS` implicitly and users should not be required to provide the project id.

Comment (mayurkale22) [reply opencensus/metrics-quickstart.js]: Yes, it is required. Will open an issue and modify example later. Is that ok?

**Pull request merged**